### DISCUSSION TITLE
Data Not Populating Properly

### DISCUSSION THREAD

**Discussion by highschooldropout** [umami-software/umami#1194] [Q&A]
Over the course of the last two releases, data has not been populating properly for me. Sometimes, it takes a while; other times, it doesn't load at all and I'm presented with an error. Umami just also seems a bit buggy and unresponsive in certain instances. Looking through other posts, it seems this isn't happening to others. Any help?

**Comment by mikecao** [comment]
Will need more information. At least an error message or something.

**Reply by highschooldropout** [reply]
Sorry about that. It seems to have been a problem with my setup and/or a caching issue. It's been resolved.
